purse

```diff
@@ -4,11 +4,12 @@ TheLortex:
   - I have 9 Gitcoins
 
 maiste:
-  - I also have 10 Gitcoins
+  - I also have 12 Gitcoins
 
 MisterDA:
   - I have drawn a beautiful camel
   - I have 11 Gitcoins
 
 CraigFe:
-  - I have 10 Gitcoins
+  - I have 8 Gitcoins
```

Comment:
```suggestion
  - I have 9.5 Gitcoins
```
I'm sorry to bother you with this change, but owning @maiste user for 2 Gitcoins is an unfair price. To consider this transaction valide, you need to put a fair price: I suggest 0.5 Gitcoin

Reply:
I see your offer of 0.5 Gitcoins and raise you `(if P == NP then 0.5 else 2)` Gitcoins.

I suggest you accept before I manage to merge a "_Githereum balances can be computationally undecidable._" clause.

Reply:
I see you are hard in business @CraigFe! @TheLortex, as a wise Githereum dictator should have the last word about it.

Reply:
Interesting.
Computationally indecidable balances seems to be great addition to the blockchain. So the _math checks out_ rule is extended to _math has to check out in any possible universe_, meaning that balances have to stay positive in both the P == NP and P != NP case. Balances are collapsed when the problem is solved, using a Proof of Proof mechanism.

Reply:
@CraigFe Please take a look at https://github.com/TheLortex/githereum/pull/19 and update the PR accordingly to add the conditional balances.

Reply:
I resolve this conf as it is updated in https://github.com/TheLortex/githereum/pull/18#discussion_r866932817